tests/unit/query/odata-adapter-test.js

```diff
@@ -800,6 +811,17 @@ test('adapter | odata | false predicate | details predicate', function (assert)
   runTest(assert, builder, 'EmberFlexberryDummyComments', `$filter=UserVotes/all(f:false)&$select=__PrimaryKey`);
 });
 
+test('adapter | odata | false predicate | details predicate', function (assert) {
```

Comment:
Для предыдущего теста можно тоже имя уточнить.
```suggestion
test('adapter | odata | false predicate | details predicate any', function (assert) {
```